src/iframeResizer.contentWindow.js

```diff
@@ -468,6 +468,30 @@
 		return maxBottomVal;
 	}
 
+	function getLowestVisibleElementHeight() {
+		var
+			allElements       = document.querySelectorAll('body *'),
+			allElementsLength = allElements.length,
+			maxBottomVal      = 0,
+			timer             = new Date().getTime();
+
+		for (var i = 0; i < allElementsLength; i++) {
+			var element = allElements[i];
+			if (element.offsetWidth === 0 || element.offsetHeight === 0) continue;
```

Comment:
this line is the crux of the thing, and the only thing i changed from the function above, `getLowestElementHeight()`